src/DefiningWANReplication.md

```diff
@@ -49,24 +58,36 @@ Following are the definitions of configuration elements:
 - `endpoints`: IP addresses of the cluster members for which the WAN replication is implemented.
 - `group.password`: Configures target cluster's group password.
 
-
+Other relevant properties are : 
+- `discovery.period` : The period in seconds in which WAN tries to reestablish connections to failed endpoints. Default is 10 (seconds).
+- `executorThreadCount` : The number of threads that the WanBatchReplication executor will have. The executor is used to send WAN events to the endpoints and ideally you want to have one thread per endpoint. If this property is omitted and you have specified the `endpoints` property, this will be the case. If necessary you can manually define the number of threads that the executor will use. Once the executor has been initialised there is thread affinity between the discovered endpoints and the executor threads - all events for a single endpoint will go through a single executor thread, preserving event order. 
```

Comment:
It has been described in the javadoc. Please check, I will add it here if it is ok : https://github.com/hazelcast/hazelcast-enterprise/blob/c06bc681ac58c958b4ce0ead172ea6e225fd1e2f/hazelcast-enterprise/src/main/java/com/hazelcast/enterprise/wan/replication/WanReplicationProperties.java#L78